qutip/qip/circuit.py

```diff
@@ -45,46 +47,50 @@ class Gate(object):
     """
     Representation of a quantum gate, with its required parametrs, and target
     and control qubits.
+    
+    Parameters
+    ----------
+    name : String
+        Gate name.
+    targets : List or Integer
+        Gate targets.
+    controls : List or Integer
```

Comment:
Integer might not be defining an object (but I might be wrong). Maybe `int` is more appropriate? But maybe also with a float that is an integer, the function works.